src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,356 @@
+
+pi: [toc, sortrefs, symrefs, comments]
+smart_quotes: off
+
+area: security
+author:
+  -
+    ins: N. Kramer
+    name: Noah Kramer
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: nkramer@ripple.com
+    uri: https://www.ripple.com
+  -
+    ins: I. Simpson
+    name: Ian Simpson
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: isimpson@ripple.com
+    uri: https://www.ripple.com
+
+normative:
+    RFC2119:
+    RFC2818:
+    RFC6265:
+    RFC7231:
+    RFC7413:
+    RFC6570:
+    RFC8446:
+    PAYID-URI:
+      title: "The 'payid' URI Scheme"
+      target: https://tbd.example.com/
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+    PAYID-DISCOVERY:
+      title: "PayID Discovery"
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+
+informative:
+    RFC5988:
+
+--- note_Feedback
+
+This specification is a draft proposal, and is part of the [PayID Protocol](https://payid.org/) initiative. Feedback related to this document should be sent in the form of a Github issue at: https://github.com/payid-org/rfcs/issues.
+ 
+--- abstract
+This specification formalizes how a payment recipient, such as a merchant or a non-profit, can automatically 
+initiate a payment from a payer using only the payer's PayID. 
+
+--- middle
+
+# Terminology
+
+This protocol can be referred to as the `PayId Easy Checkout Protocol`. It uses the following terminology:
+   
+* PayID client: a client that queries a PayID server using the PayID Protocol as defined in [PAYID-PROTOCOL][].
+* PayID server: the endpoint that returns payment account(s) information, which conforms to the PayID Protocol.
+* Merchant: Individual or entity receiving a payment (e.g., e-commerce merchant, charity).
+* Payer: Individual or entity originating a payment to a `merchant`.
+* Wallet: A device or application that holds funds (may be a non-custodial wallet).
+* PayID Easy Checkout URL: The URL that is the result of the PayID Easy Checkout protocol; can be used to redirect a client to a wallet corresponding to a particular PayID.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol that allows an online merchant to
+request a payment from a payer using only the payer's PayID. Implementations
+of the protocol should require little to no server-side engineering efforts, while creating an improved and uniform
+user experience for payers.
+
+The main focus of the Protocol is on PayID Easy Checkout Discovery, which defines how a PayID client can use a PayID
```

Comment:
I think we could strike this whole paragraph and let the protocol do the work of saying what it does. Thoughts?

Reply:
My goal here was to give a brief overview of the rest of the paper, since it's the introduction section.  The actual "RFC-y" part of PayID Easy Checkout Discovery doesn't necessarily give much context or a high level view of what it is, so I figured I'd do that in the intro.  I don't think this paragraph is busying the spec or anything either, so I would be fine leaving it in.  Happy to hear pushback to that